packages/libraries/core/src/components/file/ods-file-default-attributes.ts

```diff
@@ -0,0 +1,14 @@
+import { OdsFileAttributes, OdsFileI } from './ods-file-attributes';
+
+export const odsFileDefaultAttributesDoc  = {
+  disabled: false,
+  errorMessage: '',
+  acceptedTypes: '',
+  files: [] as OdsFileI[],
+  maxFiles: 10,
+  headerTitle: 'Glisser-déposer une pièce jointe',
+  dividerLabel: 'ou',
+  selectFilesLabel: 'sélectionner un fichier'
```

Comment:
To Be Discussed: do we set defaut french value or move from ods every text/label?